MoEPluginBase/Manager/MoEPluginInitializer.h

```diff
@@ -0,0 +1,23 @@
+//
+//  MoEPluginInitializer.h
+//  MoEPluginBase
+//
+//  Created by Chengappa C D on 14/02/20.
+//  Copyright © 2020 MoEngage. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UserNotifications/UserNotifications.h>
+#import <UIKit/UIKit.h>
+#import <MoEngage/MoEngage.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MoEPluginInitializer : NSObject
+
++ (instancetype)sharedInstance;
+- (void)initializeDefaultInstanceWithConfig:(MOSDKConfig*)sdkConfig withSDKState:(BOOL)sdkState andLaunchOptions:(NSDictionary*)launchOptions;
```

Comment:
Same here:
Change the method names to be in line with the MoEngage interface. I  believe this was discussed in the previous call.
`initializeInstanceWithConfig`
`initializeDefaultInstanceWithConfig`